libs/domains/service-helm/feature/src/lib/values-override-arguments-setting/values-override-arguments-setting.tsx

```diff
@@ -0,0 +1,214 @@
+import { type PropsWithChildren, useState } from 'react'
+import {
+  Controller,
+  type UseFieldArrayRemove,
+  type UseFormReturn,
+  useFieldArray,
+  useFormContext,
+} from 'react-hook-form'
+import {
+  Button,
+  CodeEditor,
+  EmptyState,
+  Heading,
+  Icon,
+  IconAwesomeEnum,
+  InputSelectSmall,
+  InputTextSmall,
+  Popover,
+  Section,
+} from '@qovery/shared/ui'
+
+export type ArgumentTypes = 'json' | 'generic' | 'string'
+
+export interface HelmValuesArgumentsData {
+  arguments: {
+    key: string
+    type: ArgumentTypes
+    value: string
+    json?: string
+  }[]
+}
+
+export interface ValuesOverrideArgumentsSettingProps extends PropsWithChildren {
```

Comment:
```suggestion
export interface ValuesOverrideArgumentsSettingProps extends PropsWithoutChildren {
```

Reply:
It's correct with `PropsWithChildren`? `PropsWithoutChildren` doesn't exist